source/api_details.html.erb

```diff
@@ -1,57 +1,72 @@
-<h1 id="<%= api.name.parameterize %>"><%= api.name %></h1>
+<h1 id="<%= api.name.parameterize %>">
+  <% if api.name.include? "API" %>
+    <%= api.name %>
+  <% else %>
+    <%= api.name + " API" %>  
+  <% end %>
+</h1>
+
+<% if api.description.present? %>
+  <%= render_markdown(unescape_newlines(api.description)) %>
```

Comment:
This is leading to the [TFL Unified API](https://www.api.gov.uk/tfl/transport-for-london-unified-api/#transport-for-london-unified-api) looking a bit odd - on this branch we see:

> Append the app _id and app_ key

Whereas on the current site:

> Append the `app_id` and `app_key`

Reply:
Also see `/nhs/alerts-hl7-v3-api/` as we're missing the Markdown rendering